src/chains/celo/utils.ts

```diff
@@ -66,3 +71,7 @@ export function isCIP64(
     isEmpty(transaction.gatewayFeeRecipient)
   )
 }
+
+export function generateRandomAddress(): Address {
+  return `0x${randomBytes(20).toString('hex')}` as Address
+}
```

Comment:
this should probably not be there, maybe a test-utils file ?

Reply:
absolutely right, somehow I thought it was test utils file, but it's actually not